pos_picking_load/tests/test_pos_picking_load.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+# Copyright (C) 2017: Opener B.V. (https://opener.amsterdam)
+# @author: Stefan Rijnhart <stefan@opener.am>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestPosPickingLoad(TransactionCase):
+    def test_pos_picking_load(self):
+        product = self.env.ref('product.product_product_24')
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.env.ref('base.res_partner_1').id,
+            'order_line': [(0, 0, {
+                'product_id': product.id,
+                'price_unit': 6,
+                'product_uom_qty': 2,
+            })],
+            'order_policy': 'picking',
+        })
+        sale_order.action_button_confirm()
+        self.assertTrue(sale_order.picking_ids)
+        sale_order.picking_ids.picking_type_id.write({
+            'available_in_pos': True})
+
+        config = self.env.ref('point_of_sale.pos_config_main').copy()
```

Comment:
I don't want to run into problems with sessions that are already in use (when I am performing manual tests on the same database).